gapic-common/lib/gapic/paged_enumerable.rb

```diff
@@ -95,7 +95,7 @@ def each
     #
     # @yield [Page] Gives the pages in the stream.
     #
-    # @raise [GapicError] if it's not started yet.
+    # @raise if it's not started yet.
```

Comment:
It's not possible to specify a GRPC error class? Or even just `StandardError`?

Reply:
Joe asked about this in September. I don't know what error is raised here. We will have to figure that out. But there are no tests for it either. I'm gonna leave it off for now.